### DISCUSSION TITLE
Doesn't connect to Wi-Fi network

### DISCUSSION THREAD

**Discussion by AleksNet** [arendst/Tasmota#14813] [Q&A] [answered]
Good evening!
I tried today to flash NodeMCU with the latest available firmware (Tasmota 11.0.0 by Theo Arends) and found that after setting up the network connection, the issuer: "Network connection error. Please, chesk your credentials". Although the credentials are entered correctly.This was not observed in previous versions.

**Comment by ascillato2** [comment, marked as answer]
Hi,

I have just tested with a blank NodeMCU and flashed Latest Tasmota using https://tasmota.github.io/install/ and it worked without issues.
Can you share a little more information about what you did and how you did it?
* how did you flashed your device?
* Did you erased before flashing?
* have you used the precompiled bins or selfcompile?
* are you sure that your credentials are ok?
* Are you using wifi in mode 11n-only?
* which brand is your router?
* Do you have some logs from the serial?

**Reply by sfromis** [reply]
Your credentials may not be following the stipulation about not using special characters. How this is handled may not be consistent across releases.

**Reply by AleksNet** [reply]
Hey!
First I flashed through NodeMCU PyFlasher.
Then I flashed through esptool.
Before flashing, I erased the memory: esptool.py --port /dev/ttyUSB0 -b 115200 erase_flash
Then after that I already flashed it: esptool.py --port /dev/ttyUSB0 write_flash -fs 1MB -fm dout 0x0 tasmota.bin
I took the firmware file here: http://ota.tasmota.com/tasmota/release/
About the fact that I'm sure that my credentials are in order. Answer: Yes. Since just yesterday I was flashing Sonoff T4EU1C. True, for some reason, the firmware version yesterday was completely different (Tasmota 10.1.0 by Theo Arends), downloaded from the same link as above.
The router uses several modes: b+g+n
Router: TP Link
Unfortunately there are no magazines.

**Reply by ascillato** [reply]
Try to flash from https://tasmota.github.io/install/

From that flasher, you can see the logs too.
